docs/cpan-dependency-confusion.md

```diff
@@ -225,10 +225,19 @@ Use reserved namespaces that can't resolve to public things:
 
 From [PAUSE Naming Modules - Local::](https://pause.perl.org/pause/query?ACTION=pause_namingmodules#Local) and [The Perl Module Library (perlmodlib)](https://perldoc.perl.org/perlmodlib#Guidelines-for-Module-Creation)
 
+#### Namespace vacuum
+The CPAN ecosystem is not vulnerable to "Namespace vacuum" as described in [this article](https://cybersecuritynews.com/popular-python-library-vulnerability/) (python/PyPI).
+
+Indeed for the following reasons: 
```

Comment:
I find the phrase "Indeed for the following reasons:" a bit odd...

(though this might be not being exposes to the wording, rather than anything about it's correctness)

Would it be clearer by using the phrase "The reasons for this are:" ?